### PR TITLE
initial commit for PR

### PR DESCRIPTION
**Pull request by duhbhavesh**
Hello, Please review my PR.

Please ignore ```.map, .svg, .css, prism``` files.

Thank you!

### CODE REVIEW

File: docs/documentation.html [hunk 1]
```diff
@@ -0,0 +1,1210 @@
...
+				</nav>
+				<button
+					onclick="scrollToTop()"
+					class="btn btn-square btn-primary btn-circle btn-fab btn-fab-arrow"
+				>
+					<i class="fas fa-angle-double-up"></i>
+				</button>
+			</div>
+		</header>
+		<aside class="container sidebar">
```
Comment (shubham-singh) [inline docs/documentation.html]: sidebar is not scrollable on desktop

try adding
 
**overflow-y: auto;**

in docs.css

File: css/docs.css [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    width: 75%;
+  }
+}
+@media (min-width: 1280px) {
+  .sidebar {
+    position: fixed;
+    min-width: 250px;
+    left: 80px;
+    width: 200px;
+  }
```
Comment (shubham-singh) [inline css/docs.css:145]: min-width is greater than the provided width for some reason,

File: css/drift.css [hunk 3]
```diff
@@ -0,0 +1,1180 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
+/* Document
+   ========================================================================== */
+/**
+ * 1. Correct the line height in all browsers.
+ * 2. Prevent adjustments of font size after orientation changes in iOS.
+ */
```
Comment (shubham-singh) [inline css/drift.css:7]: try to use css variables as much as possible. variables will give your component library uniformity

File: css/drift.css [hunk 4]
```diff
@@ -0,0 +1,1180 @@
...
+  flex-direction: column;
+  align-items: flex-start;
+  margin-top: 0.8rem;
+}
+.lists .list {
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  border-radius: 0.4rem;
+  box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
```
Comment (shubham-singh) [inline css/drift.css:1002]: avoid using pixels. maybe try em or rem.

**Review by shubham-singh — COMMENTED** [review]
In usability, the navbar on desktop is not scrollable.

In your css, try to use variables.

Drift CSS looks awesome overall.

File: css/drift.css (hunk 3 above)
Comment (duhbhavesh) [reply css/drift.css:7]: It is compiled css, I'm using sass

Comment (shubham-singh) [reply css/drift.css:7]: aaah ok. then its ok.

i have not used sass, i am guessing it is replacing your variables into values after compilation?

Comment (duhbhavesh) [reply css/drift.css:7]: Yes, correct✌ @shubham-singh

**Comment by duhbhavesh** [comment]
Hey, @shubham-singh I've made the necessary changes please look into it. -https://github.com/duhbhavesh/driftcss/pull/1/commits/6b26a02399c9d7721db555ea59a56a2fa29d0d04

**Review by shubham-singh — COMMENTED** [review]
Looks all good Bhavesh!

**Review by manan228 — COMMENTED** [review]
Looks good

**Comment by duhbhavesh** [comment]
Merging PR from ```development``` to ```main```

**Comment by jay-rajput30** [comment]
looks good to merge

**Pull request merged**